crypto/.js/src/main/scala/org/bitcoins/crypto/ECDSA.scala

```diff
@@ -41,11 +41,21 @@ class ECDSA(
 
   def verify(msg: Buffer, sig: Buffer, key: Buffer): Boolean = js.native
 
+  def signDER(msg: Buffer, key: Buffer): Buffer = js.native
+
+  def verifyDER(msg: Buffer, sig: Buffer, key: Buffer): Boolean = js.native
+
   def recover(
       msg: Buffer,
       sig: Buffer,
       param: Byte,
       compress: Boolean): Buffer = js.native
 
+  def recoverDER(
+      msg: Buffer,
+      sig: Buffer,
+      param: Byte,
+      compress: Boolean): Buffer = js.native
```

Comment:
Seeing as DER is the default in Bitcoin, I find it concerning to have `recover` and `verify` and `sign` not be doing DER-related things?